EIPS/eip-725.md

```diff
@@ -1,55 +1,93 @@
----
+```
 eip: 725
-title: Proxy Account
-author: Fabian Vogelsteller (@frozeman), Tyler Yasaka (@tyleryasaka)
+title: ERC-725 Smart Contract Based Account
+author: Fabian Vogelsteller <fabian@lukso.network>, Tyler Yasaka (@tyleryasaka)
 discussions-to: https://github.com/ethereum/EIPs/issues/725
 status: Draft
 type: Standards Track
 category: ERC
+requires: ERC165, ERC173, ERC1271 (optional)
```

Comment:
```suggestion
requires: 165, 173, 1271
```
Frontmatter is *very* strict on formatting.

Reply:
donaldsullender@gmail.com

Reply:
I think having the ERC is important as there will be other standards proposals from other chains, that are relevant to Ethereum smart contracts as well. In LUKSO we use `LSP` (LUKSO standard proposals), that will heavily extend 725

Reply:
If you feel strongly about this, I recommend bringing it up to the EIP editors in Discord or via a pull request to EIP-1 and the validator bot.  In the meantime, I recommend making this suggested change if you want to get this merged.